### ISSUE TITLE
Format using Black

### ISSUE THREAD

**Issue by jmp** [jmp/farbfeld#3]
Use [Black](https://github.com/ambv/black/) to format the code.

**Comment by jmp** [comment]
On second thought, Black seems to produce some undesirable results in certain cases.

**Case 1:**
```python
rows.append(pixels[offset:offset + num_rows])
```
becomes
```python
rows.append(pixels[offset : offset + num_rows])
```
which actually seems to violate PEP-8: `farbfeld.py:134:34: E203 whitespace before ':'`

**Case 2:**
```python
validate_pixel = functools.partial(
    _validate_items,
    COMPONENT_NUM,
    _validate_component,
)
```
becomes
```python
validate_pixel = functools.partial(
    _validate_items, COMPONENT_NUM, _validate_component
)
```
which is less readable than the original.

So, abort mission for now.

**Comment by jmp** [comment]
Reconsidered and resolved in d29c41b26a31dd83a6e63bd24babeb35f20c545e.
